packages/atomic/cypress/utils/network.ts

```diff
@@ -17,3 +17,17 @@ export function getAnalytics(selector: string): Promise<any> {
     });
   });
 }
+
+export function getApiResponseBodyAt(selector: string, order: number) {
+  for (let i = 0; i < order; i++) {
+    getApiResponseBody(selector);
+  }
+  return getApiResponseBody(selector);
+}
+
+export function getAnalyticsAt(selector: string, order: number) {
+  for (let i = 0; i < order; i++) {
+    getAnalytics(selector);
+  }
+  return getAnalytics(selector);
+}
```

Comment:
Hmm, these two functions look strange. If an `order` > 0 is passed, they call a function, and then call it one more time at the end and return the value?

Since only the last function call is returned, are the earlier calls inside of the loop useful?

Reply:
if search page returns only 1 UA, then cypress will capture it correctly with only 1 time call `getAnalytics()`
but if it returns more than 1, first  is only able `getAnalytics()`  to capture the 1st UA sent. So we need to repeat the call `getAnalytics()` to capture the 2nd/3rd UA 

In this case, because pager-component won't generate with we don't execute search when loading searchpage, so I have to executeSearch from first load, and the first UA is already `interfaceLoad`. Therefore, we need to call the second time to capture next UA of all the clicks/searches that we want to validate.